samcli/lib/cookiecutter/interactive_flow_creator.py

```diff
@@ -42,6 +42,14 @@ def create_flow(flow_definition_path: str, extra_context: Optional[Dict] = None)
                         "True": "key of the question to jump to if the user answered 'Yes'",
                         "False": "key of the question to jump to if the user answered 'Yes'",
                       }
+                      "default": "default_answer",
+                      "default_from_toml": {
```

Comment:
Can we document the format somewhere? It's not clear what `section`/`cmd_names`/`key` are.

Reply:
Also minor nit: let's use consistent capitalization everywhere. Since we have `nextQuestion`, rename to `defaultFromToml` etc.